.jenkins/dsl/jobs.groovy

```diff
@@ -18,6 +18,27 @@ def getDefaultJobParams(String repoName = 'optaplanner') {
             repository: repoName,
             credentials: "${GIT_AUTHOR_CREDENTIALS_ID}",
             token_credentials: "${GIT_AUTHOR_TOKEN_CREDENTIALS_ID}"
+        ],
+        env: [:],
+        pr: [:]
+    ]
+}
+
+Map getMultijobPRConfig() {
+    return [
+        jobs : [
```

Comment:
What one has to do in order to add more repositories downstream, say optaplanner-quickstarts and both optawebs?

Reply:
You are right, this is missing some documentation... gonna add it.